ci/ansible/playbooks/os/redhat/leapp_metadata.yml

```diff
@@ -0,0 +1,11772 @@
+$ANSIBLE_VAULT;1.1;AES256
+37373839313765333931663935386630383835303064633931396365613830343961396634306531
+3935333430373638623061316437376636363838613863340a323731336435636538653562356265
+39363033373735346639383132643565633035306563616635333831353166343664633063643566
+3663323937363161340a633462616365313863663639313937346462666535316264366162643534
+39343563373461646265616136376334303830366134616239663030323639396235373234326263
+31306233646139313463333138366164616432323162386465376264653064373439373762326430
```

Comment:
We were discussing about this with @to-bar and we have doubts if this should be even in encrypted form in our repository because of the licensing. Probably we should remove this?

Reply:
Removed from repo, provided via Azure File share to build agents.